### PR TITLE
profiles: mov-cli: remove ffmpeg & allow more paths

### PR DESCRIPTION
**Pull request by pirate486743186**
Changes:

* Remove ffmpeg from private-bin
* Allow download folder
* It needs an editor to allow editing the config, so I put in nano; sh
  and uname are used for launching nano

### CODE REVIEW

File: etc/profile-m-z/mov-cli.profile [hunk 1]
```diff
@@ -24,9 +29,9 @@ noprinters
 notv
 
 disable-mnt
-private-bin ffmpeg,fzf,mov-cli
+private-bin fzf,mov-cli,nano,uname,sh
```
Comment (glitsj16) [inline etc/profile-m-z/mov-cli.profile:32]: sorting: private-bin fzf,mov-cli,nano,sh,uname
Probably why checks failed: https://github.com/netblue30/firejail/actions/runs/8597879679/job/23557480018?pr=6304

File: etc/profile-m-z/mov-cli.profile [hunk 2]
```diff
@@ -24,9 +29,9 @@ noprinters
...
 #private-cache
-private-etc X11,alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,group,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,magic,magic.mgc,mime.types,nsswitch.conf,pango,passwd,pki,protocols,pulse,resolv.conf,rpc,services,ssl,xdg
+private-etc X11,alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,group,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,magic,magic.mgc,mime.types,nsswitch.conf,pango,passwd,pki,protocols,pulse,resolv.conf,rpc,services,ssl,xdg,nanorc
```
Comment (glitsj16) [inline etc/profile-m-z/mov-cli.profile:34]: sorting: ...,mime.types,nanorc,nsswitch.conf,...
Probably why checks failed: https://github.com/netblue30/firejail/actions/runs/8597879679/job/23557480018?pr=6304

**Review by glitsj16 — CHANGES_REQUESTED** [review]
LGTM; needs changes to pass CI sorting checks.

**Review by glitsj16 — APPROVED** [review]

**Comment by glitsj16** [comment]
> wait... sort.py is broken? It doesn't actually edit the file, but it tells you the correct edits.

sort.py isn't broken, but there were recent changes you might have missed: https://github.com/netblue30/firejail/commit/aa08aa132bcc48f550fa354c28877753ff2679a3. You'll have to use the `-i` option from now on to write the edits.

File: etc/profile-m-z/mov-cli.profile [hunk 3]
```diff
@@ -24,9 +29,9 @@ noprinters
 notv
 
 disable-mnt
-private-bin ffmpeg,fzf,mov-cli
+private-bin fzf,mov-cli,nano,sh,uname
```
Comment (kmk3) [inline etc/profile-m-z/mov-cli.profile:32]: Why remove ffmpeg from private-bin?

**Review by kmk3 — APPROVED** [review]

**Pull request merged**